pkg/microservice/aslan/core/workflow/service/workflow/pipeline_task.go

```diff
@@ -930,15 +934,17 @@ func GetArtifactFileContent(pipelineName string, taskID int64, log *zap.SugaredL
 				return nil, fmt.Errorf("failed to create file %s %v", artifactFileName, err)
 			}
 
-			defer func() {
-				_ = file.Close()
-				_ = os.Remove(path.Join(sourcePath, artifactFileName))
-			}()
-
-			err = s3.Download(context.Background(), s3Storage, artifactFile, file.Name())
+			client, err := s3tool.NewClient(s3Storage.Endpoint, s3Storage.Ak, s3Storage.Sk, s3Storage.Insecure)
```

Comment:
download files concurrently

Reply:
We will do this in another PR, this is not a feature for this time